lib/valueflow.cpp

```diff
@@ -1221,17 +1221,37 @@ static void valueFlowSameExpressions(TokenList *tokenlist)
     }
 }
 
+static bool isIncompleteVar(const Token *tok)
+{
+    if (!Token::Match(tok, "%name%"))
+        return false;
+    if (tok->tokType() != Token::eName)
+        return false;
+    if (tok->flags() != (1 << 20))
```

Comment:
Hmm.. I do not really like this `1 << 20`, it is too hardcoded. I would like to be able to insert a Token flag.